dev/tools/gen_keycodes/README.md

```diff
@@ -58,109 +58,62 @@ the API.
+plane.  For example, plane 0x1 refers to the range 0x1 0000 0000 -
+0x1 FFFF FFFF.  Each plane manages how the values within the range are assigned.
 
-- **0x00 0000 0000 - 0x0 0010 FFFF**: For keys that generate Unicode
+The planes are planned as follows:
+
+- **Plane 0x00**: The Unicode plane. This plane contains keys that generate Unicode
   characters when pressed (this includes dead keys, but not e.g. function keys
-  or shift keys), the logical key code is the Unicode code point corresponding
-  to the representation of the key in the current keyboard mapping. The
-  Unicode code point might not match the string that is generated for
-  an unshifted keypress of that key, for example, we would use U+0034 for the
-  “4 \$” key in the US layout, and also the “4 ;” key in the Russian layout,
-  and also, maybe less intuitively, for the “' 4 {“ in French layout (wherein
-  the latter case, an unshifted press gets you a ', not a 4). Similarly, the Q
-  key in the US layout outputs a q in normal usage, but its code would be 0x0
-  0000 0051 (U+00051 being the code for the uppercase Q).
-
-- **0x01 0000 0000 - 0x01 FFFF FFFF**: For keys that are defined by the [USB HID
-  standard](https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf),
-  the key code consists of the 32 bit USB extended usage code. For
-  example, the Enter key would have code 0x01 0007 0028. Only keys that fall
-  into collections "Keyboard", "Keypad", and "Tablet PC System Controls" are
-  considered for this API; for example, a mixing desk with multiple
-  collections of volume controls would not be exposed via DOWN and UP events,
-  nor would a mouse, joystick, or golf simulator control.
-
-- **0x02 0000 0000 - 0xFF FFFF FFFF**: For keys that aren't defined in USB at the
-  time of implementation, but that we need to support. For example, if Flutter
-  were ever ported to the Symbolics LM-2, the "thumb up" key might be given
-  the code 0x14 0000 0001, where 0x14 is defined as the “Symbolics” platform
-  range. Where possible, we will use specific subranges of this space to reuse
-  keys from other platforms. When this is not possible, the prefix 0xFF is
-  reserved for “Custom” codes. Each platform from which we take codes will get
-  a unique prefix in the range 0x2-0xFE. If multiple systems define keys with
-  the same usage (not the same number), then the value with the lowest prefix
-  is used as the defining code.
-
-  Prefixes will be:
+  or shift keys). The value is defined as the Unicode code point corresponding
+  to the character, lower case and without modifier keys if possible.
+  Examples are Key A (0x61), Digit 1 (0x31), Colon (0x3A), and Key Ù (0xD9).
+  (The "Colon" key represents a keyboard key that prints the ":"
+  character without modifiers, which can be found on the French layout.  On the
+  US layout, the key that prints ":" is the Semicolon key.)
+  This plane also contains key None (0x0).
```

Comment:
Honestly, I don't know. I kept it for 2 reasons:
- It was there (because `dom_code_data` contained it as a scancode).
- It is a special value, 0 (which is also why I kept it in the 0x000 plane.)

Do you think I should remove it?

Reply:
Removed `None` from both logical and physical.